### ISSUE TITLE
Autocomplete for adding a child work relationship never returns any works

### ISSUE THREAD

**Issue by lfarrell** [projecthydra/sufia#3129]
If you start typing a work title in the relationships->add child text box of a work the autocomplete feature never returns any works to select from. It always returns an empty array.

There's no stack trace or obvious error generated.

**Comment by mjgiarlo** [comment]
@lfarrell To confirm, does typing in the ID work?

**Comment by lfarrell** [comment]
@mjgiarlo Yes, it appears to create the relationship, but then it hits the other issue of not being able to save the record.

**Comment by mjgiarlo** [comment]
In case it helps w/ analysis... this was worked on by @andjsmit here: https://github.com/projecthydra/sufia/commit/798495543e99cf84707dba64fe45de150e9548b7

**Comment by lfarrell** [comment]
@mjgiarlo Okay, I'll try and take a look.

**Comment by andjsmit** [comment]
@lfarrell @mjgiarlo My first guess would be that nothing is being returned due to the "can you edit it?" filter. So if the issue with the locked files is solved, I wonder if the autocomplete will come back.

**Comment by jcoyne** [comment]
This seems to be working for me:

<img width="679" alt="screen shot 2017-03-10 at 10 52 22 pm" src="https://cloud.githubusercontent.com/assets/92044/23820709/664161ce-05e4-11e7-83e9-6d79883f86d4.png">

Then when I select one an push "add":
<img width="729" alt="screen shot 2017-03-10 at 10 52 53 pm" src="https://cloud.githubusercontent.com/assets/92044/23820712/6d4db5a8-05e4-11e7-97a0-4eee402e1c0f.png">
